code/style.css

```diff
@@ -40,19 +57,26 @@ p {
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: #000066;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: white;
   border: none;
   font-size: 20px;
   border-radius: 8px;
   display: block;
   margin: 0 auto;
 }
+
+#message {
```

Comment:
Better to style h3 or class than id in css? Am not sure :-)

Reply:
Definitely, that's an antipattern to use id for styling